package_installs.R

```diff
@@ -1,7 +1,9 @@
 # Repo to pull package data and metadata from.
 REPO <- 'http://ftp.osuosl.org/pub/cran'
 # Number of parallel installs.
-M <- 4
+M <- 64
```

Comment:
Leave a comment as to why this is not 32 (if that is intended).  Possibly, this should be a factor of the 32 / amount of CPUs in the VM?

Reply:
So I've done a few manual testing for that parameter.
- If set too high (128), it crashes.
- `64` is slightly faster than `32`